### PR TITLE
Fix MSVC Windows builds (and broken IRC links)

### PR DESCRIPTION
**Pull request by programmer2514**

### CODE REVIEW

**Comment by adriweb** [comment]
Looks good to me, the dynamic builds were successful: https://github.com/CE-Programming/CEmu/actions/runs/1835120905
Let's just wait for @alberthdev's review I guess? Unless someone else says it's OK.

Anyway, the static builds need to be fixed or removed...
In the meantime non-msvc static builds are available on https://jacobly.com/CEmu/master/CEmu64.exe

**Comment by alberthdev** [comment]
> Anyway, the static builds need to be fixed or removed... 

Haven't quite gotten back to it yet, but we would need another fresh build of static Qt to get things going again. (It's still a WIP, somewhat in hiatus...) Totally fine with disabling the static builds for now (if that's easy to do without completely removing them) until that's back in place again!

**Comment by adriweb** [comment]
I suppose you should clean your PR commits by rebasing?

**Comment by adriweb** [comment]
You should probably talk to us on the chan (#cemu-dev on IRC EFNet, or on Discord, the link is in the readme or maybe the toolchain's readme), it'll be simpler to organize the resting of your commits. 
BTW you should `git rebase master` your branch instead of merging to keep it up to date, if you want a cleaner commit history.

**Comment by programmer2514** [comment]
Okay, will do. I was just messing around (probably should've made a new branch :/) and was planning to rebase once I actually did anything that worked. I'll hop into the Discord though, sorry for the mess

File: .github/workflows/build.msvc.windows.workflow.yml [hunk 1]
```diff
@@ -139,7 +140,7 @@ jobs:
       shell: cmd
       run: |
         echo Creating directory to prevent failure: vcpkg\downloads
-        md vcpkg\downloads
+        if not exist vcpkg\downloads md vcpkg\downloads
```
Comment (alberthdev) [inline .github/workflows/build.msvc.windows.workflow.yml:143]: This makes sense and prevents setting %ERRORLEVEL%. One thing I also found is this - not sure if worth considering too: https://stackoverflow.com/a/4165472

**Review by alberthdev — APPROVED** [review]
Looks good to me so far! One minor comment re: an edge casse that might matter. Might be worth running a CI on this PR to see how things turn out.

File: .github/workflows/build.msvc.windows.workflow.yml [hunk 2]
```diff
@@ -102,13 +103,15 @@ jobs:
...
-        pip install requests jinja2 irc
+        pip install requests jinja2 irc configargparse
     
     # Install our static dependencies via helper script
     # TODO: get caching working here too
     - name: Install static dependencies via bootstrap helper script
       if: ${{ matrix.linking == 'static' }}
-      run: python -u gui\qt\deploy-scripts\appveyor_helper.py install
+      run: |
+        python -u gui\qt\deploy-scripts\BuildQtDevArchive.py --config gui\qt\deploy-scripts\qt_dev_archive.static.x86.cfg --qt-version ${{ env.QT_VERSION }}
```
[the remark this answers is not in the document]
Comment (adriweb) [reply .github/workflows/build.msvc.windows.workflow.yml]: Hmm did you comment on an old version of the PR? The latest version doesn't change a whole lot of things so far, but I guess the main point for now is to get the Qt 5.15 static build done first anyway

**Comment by adriweb** [comment]
I've run the action so you can take a look at what's failing now

File: .github/workflows/build.msvc.windows.workflow.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (programmer2514) [reply .github/workflows/build.msvc.windows.workflow.yml]: Yeah, that's an old version. My latest changes can now be found in [programmer2514:qt-build-automation](https://github.com/programmer2514/CEmu/tree/qt-build-automation) until I actually get something working.

File: .github/workflows/build.msvc.windows.workflow.yml (hunk 1 above)
Comment (programmer2514) [reply .github/workflows/build.msvc.windows.workflow.yml:143]: I'm a bit confused...do you just mean to log whether or not the folder exists already?

**Comment by programmer2514** [comment]
I actually have already fixed the dynamic builds again, and am currently automating building/deploying the latest static Qt5 whenever the corresponding artifacts are missing (see below). I'll merge it into master once I actually get it working (should save you the constant approval spam, and also keep the commit history cleaner).
> Yeah, that's an old version. My latest changes can now be found in [programmer2514:qt-build-automation](https://github.com/programmer2514/CEmu/tree/qt-build-automation) until I actually get something working.

**Comment by programmer2514** [comment]
Figured this might as well not stay open with broken commits until I actually get around to fixing it

**Pull request closed without merging**